### PR TITLE
Fix typos in Portfolio workshop

### PR DESCRIPTION
**Pull request by klipspringr**

### CODE REVIEW

File: workshops/portfolio/README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ side, we see that our original text is still there:
 
 > ![](img/scroll_down.gif)
 
-Ok. We need to make the picture smaller
+Okay. We need to make the picture smaller
```
Comment (zachlatta) [inline workshops/portfolio/README.md]: Ok is actually grammatically correct (as much as I wish it wasn't) :-)

Comment (paked) [inline workshops/portfolio/README.md]: @zachlatta is there a particular reason you didn't merge this?

File: workshops/portfolio/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-## Our Mission
+## Goals
 
-Our mission:
+Goals for this workshop:
 
-- make your personal website
-- put it on the internet for the whole world to see!
+- Wake your personal website
```
Comment (MaxWofford) [inline workshops/portfolio/README.md]: s/W/M/

**Comment by klipspringr** [comment]
I *M*ade a dumb
(M, meant to fix that....)

**Comment by MaxWofford** [comment]
Moved to https://github.com/hackedu/hackedu/pull/416

**Comment by jonleung** [comment]
@klipspringr SOOOO MUCH LOOOOVE! THAAAAAAAANKS!

**Pull request closed without merging**